src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final String INDEX_FOR_MESSAGE = "Index: ";
+    private static final String SIZE_FOR_MESSAGE = ", Size:";
+
+    private Object[] elementData;
```

Comment:
saves us from casting the values on returns
```suggestion
    private T[] elementData;
```